apps/cli/vitest.config.ts

```diff
@@ -3,6 +3,7 @@ import { defineConfig } from "vitest/config";
 export default defineConfig({
     test: {
         coverage: {
+            enabled: true,
```

Comment:
Ok, ignore previous config:coverage comments

Reply:
But still, this config will only aplly to apps/cli AFAIK.

Reply:
only cli has coverage test anyway